lib/t/feeds.ex

```diff
@@ -625,4 +629,148 @@ defmodule T.Feeds do
     |> where([s], s.inserted_at < fragment("now() - ? * interval '1 day'", ^ttl_days))
     |> Repo.delete_all()
   end
+
+  # newbies and stuff
+
+  @doc """
+  Starts a "Since Live" event for newbies:
+
+    - sends a message to the admin tg bot
+    - broadcasts a push notification to all participants
+    - broadcasts a `:start` event to all newbies' feed channels
+
+  This function shouldn't be called directly, but rather scheduled with Oban in crontab.
+  """
+  @spec newbies_start_live(DateTime.t()) :: :ok
+  def newbies_start_live(reference \\ DateTime.utc_now()) do
+    newbies = newbies_list_today_participants(reference)
+
+    m = "LIVE (newbie's special edition) started: newbies count #{length(newbies)}"
+    Logger.warn(m)
+    Bot.async_post_message(m)
+
+    Enum.each(newbies, fn user_id ->
+      feed_channel_topic = "feed:" <> user_id
+      broadcast(feed_channel_topic, {__MODULE__, [:mode_change, :start]})
+    end)
+
+    newbies
+    |> T.Accounts.list_apns_devices()
+    |> DispatchJob.schedule_apns("newbie_live_mode_started", _data = %{})
+
+    :ok
+  end
+
+  @doc """
+  Ends a "Since Live" event for newbies:
+
+    - sends a message to the admin tg bot
+    - broadcasts a `:end` event to all `:mode_change` subscribers
+    - clears live tables
+
+  This function shouldn't be called directly, but rather scheduled with Oban in crontab.
+  """
+  @spec newbies_end_live :: :ok
+  def newbies_end_live do
+    m = "LIVE (newbie's special edition) ended"
+    Logger.warn(m)
+    Bot.async_post_message(m)
+
+    # we don't need to send `end` event only to newbies,
+    # we can turn off everyone who's live right now
+    # since "newbie live" and "normal live" don't intersect
+    notify_subscribers(:mode_change, :end)
+
+    two_hours_ago_ref = DateTime.add(DateTime.utc_now(), -7200)
+    clear_live_tables(two_hours_ago_ref)
+
```

Comment:
Not sure about what push notifications to send when the "newbies" version of Since Live ends. In the real Since Live the notifications say when the next Since Live is, should it be the same here, it would say when the next real Since Live is?

Reply:
> should it be the same here, it would say when the next real Since Live is?

I think so